week1-section3/exercise3/Main.cpp

```diff
@@ -0,0 +1,83 @@
+/*Exercise 2 - More Input Validation
+Rewrite the true or false quiz you did in the previous section. This time, instead of asking for 1 or 2 as inputs, ask them to answer T or F. If they enter invalid input, give them a couple of tries before you automatically fail them.
+*/
+#include <iostream>
+
+
+int main()
+{
+	std::cout << "Welcome to this quiz\How many doy you know about general culture?"<<std::endl;
+	std::cout << "Remember to use F to false or T to true\nLet's play!!\n"<<std::endl;
+
+
+	//creating asks and answers
+	std::string question[5] = {"The paramecium is a flower? ","does the platypus lay eggs? ", "Is the whale a mammal? ","Is the bat a bird? ", "Is the tomatoe a fruit? "};
+	std::string answer[5] = {"F","T","T","F","T"};
+	
+	//creating inputs
+	std::string answer2;
```

Comment:
Choose the names of your variables carefully. Don't just add a 2 to a name that is already in use, especially in this case where these aren't related at all.

Having a variable called `answer` and another called `answer2` makes it sound like there are two answers. Nothing about it makes me think that `answer` is an array of correct answers, and `answer2` is the actual input of the user.